ui/packages/consul-ui/app/components/topology-metrics/card/index.js

```diff
@@ -0,0 +1,11 @@
+import Component from '@glimmer/component';
+import { get } from '@ember/object';
+
+export default class TopologyMetrics extends Component {
+  // =methods
+  get hrefPath() {
+    const source = get(this.args.item, 'Source');
+
+    return source === 'routing-config' ? 'dc.routing-config' : 'dc.services.show.index';
+  }
+}
```

Comment:
I tried doing this in the template file, but I ran into some issues. First, there is already a ternary in the template file wrapped around the anchor tag. Secondly, adding another ternary or if/else made this look very complicated. 

>I know we've spoken quite a bit in the past about preferring doing these kinds of things straight in the template using helpers instead of in the backing component class, and here its even more preferable.

I don't want to risk having any string issues in the template. This is a cleaner approach and you can clearly see which area in the `href` I'm changing and how.